src/modules/components/proposal/proposalVoting/proposalVotingTabs/proposalVotingTabs.tsx

```diff
@@ -44,7 +44,7 @@ export const ProposalVotingTabs: React.FC<IProposalVotingTabsProps> = (props) =>
                 />
                 <Tabs.Trigger label={copy.proposalVotingTabs.details} value={ProposalVotingTab.DETAILS} />
             </Tabs.List>
-            <div className="flex grow flex-col" ref={contentRef}>
+            <div className="flex grow flex-col gap-y-2" ref={contentRef}>
```

Comment:
Which element does this spacing affect? Can you post a Figma link?

Reply:
https://www.figma.com/design/ISSDryshtEpB7SUSdNqAcw/Governance-UI-Kit?node-id=16752-19246&m=dev